Author: Cesare Falco <c.falco@ubuntu.com>
Description: Fix linking against system libraries.
 Fix linking Mame tools against zlib and flac system libraries
 instead of embedded ones.
Forwarded: yes

Index: mame/makefile
===================================================================
--- mame.orig/makefile	2015-01-29 11:46:28.852294000 +0100
+++ mame/makefile	2015-01-29 11:47:03.556294000 +0100
@@ -756,6 +756,7 @@
 ZLIB = $(OBJ)/libz.a
 else
 LIBS += -lz
+BASELIBS += -lz
 ZLIB =
 endif
 
@@ -766,6 +767,7 @@
 # $(OBJ)/libflac++.a
 else
 LIBS += -lFLAC
+BASELIBS += -lFLAC
 FLAC_LIB =
 endif
 
